integration/src/test/resources/foreach-test.xml

```diff
@@ -60,7 +60,7 @@
     </flow>
 
     <flow name="map-config">
-        <foreach>
+        <foreach collection="#[dw:dw::core::Objects::entrySet(payload)]">
```

Comment:
Why do you still need the prefix after this changes?

Reply:
missing push